lib/main.js

```diff
@@ -60,14 +59,6 @@ var defaults = {
   mmlNode: false,              // DOM element for MML output?
   svg: false,                     // return svg output?
   svgNode: false,              // DOM element for SVG output?
-
-  speakText: false,               // add spoken annotations to svg output?
```

Comment:
Thanks.  Note, however, that the comment says it adds to svg, but it really adds to all output formats.